mbed_devices/_internal/htm_file.py

```diff
@@ -69,11 +69,13 @@
 class OnlineId(NamedTuple):
```

Comment:
Not part of this PR, but I'd like to see this `NamedTuple` changed to a `dataclass(frozen=True)`

Reply:
I'm not going to do this now, but that seems like a reasonable change.